bin/dynamic_config/update_gatekeeper.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+FULL_CONFIG_PATH=`readlink -f "$1"`
+cd /home/ubuntu/adhoc/dashboard
+bundle exec ruby /home/ubuntu/dynamic_config/lib/update_gatekeeper.rb $FULL_CONFIG_PATH
```

Comment:
Is this guaranteed to be the location? Can we make this a relative path so this can be run locally?